### PR TITLE
Adds netgun, haywire rounds and gun related fixes

### LINKED ISSUE #1923: Electrified Arm does nothing
The electrified arm on a emagged engineering cyborg doesn't appear to do anything when used on other people. No message, no effect.

### PR DESCRIPTION
**Pull request by Alberyk**
-adds the netgun, replacing the syndicate cyborg energy crossbow with a downgraded mounted version, and giving the heisters a chance of getting one of those in their random gun spawn
-fix the meteor gun, giving it a new sprite and fixing the wrong icon state on the projectile
-ports emp/haywire rounds from baystation, only .38 and 12 gauge rounds
-fix the action button of the am rifle not wielding it
-gives the wizard staffs tech levels
-removes the xray crate from cargo
Fixes #1923

### CODE REVIEW

**Comment by LordFowl** [comment]
This just in on "Cheapening weapon effects by creating a variant for every gun that does it". I disapprove of giving what seems to be every major ballistic weapon a variant of Ion rifle bullets.

Also, why exactly is a ranged handcuff gun a good idea? Especially when it seems to be unimplemented? In fact, all of these are unimplemented, which makes me wonder...why?

Oh right, developer creep. Add something for "uhh, adminbus", and then sneak it in later.
Reminds me we need to port /tg/'s power crepe.

**Comment by Alberyk** [comment]
Energy nets works differently from cuffs, also, hireway rounds are mostly likely being added to research in this pr.

**Comment by LordFowl** [comment]
Energy nets work almost identically to cuffs. They trap you and prevent you from interacting with the world until you complete a breakout timer. In fact, they're even worse because they also prevent you from moving.

**Comment by Alberyk** [comment]
Nope, energy nets atm allow you still to interact with things and etc, you can't even use resist to get out of them, just by attacking the net, which you can do even if you are trapped inside of one.
https://i.imgur.com/7iGiIsO.png

**Comment by LordFowl** [comment]
So they're only slightly different from handcuffs. Similar enough that creating what is presumably going to be a security weapon out of them to use at range is still pretty ridiculous.

**Comment by Alberyk** [comment]
I changed some things based on your feedback;
-net guns are now an antag tool, being a possibility in the heist random gun spawn and the syndicate cyborgs get their own version, replacing the mounted energy crossbow
-remove the ,45 amd 99mm haywire rounds, leaving the .38 and the shotgun shell, which are already possible via cargo and zipguns

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 1]
```diff
@@ -227,8 +227,8 @@
 		/obj/item/ammo_casing/shotgun           = "12 gauge",
 		/obj/item/ammo_casing/shotgun/pellet    = "12 gauge",
 		/obj/item/ammo_casing/shotgun/pellet    = "12 gauge",
```
Comment (Arrow768) [inline code/modules/projectiles/guns/projectile/pistol.dm]: Is there a reason why some ammo types are duplicated and some are not ?

Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/pistol.dm]: No idea what was the original purpose when some bay code added it, I guess it was to make them more common than others?

**Pull request merged**